www_src/pages/project/touchhandler.js

```diff
@@ -1,69 +1,126 @@
 module.exports = function(positionable) {
-  return {
+
+  var resetTransform = function() {
+    return {
+      x1: false,
+      y1: false,
+      x2: false,
+      y2: false,
+      distance: 0,
+      angle: 0
+    };
+  };
+
+  var transform = resetTransform();
+  var mark = JSON.parse(JSON.stringify(positionable.state));
```

Comment:
I've started to bring in some single serving utilities ... perhaps we could include [lodash.clone](https://www.npmjs.com/package/lodash.clone) to make this a little easier to read?